Volume/Views/WeeklyDebriefView.swift

```diff
@@ -6,112 +6,251 @@
-                                .frame(width: 245, height: 75)
-                            Text("Stay updated with Cornell student publications, all in one place")
-                                .font(.begumRegular(size: 16))
-                        }
-                        .padding(.top, 200)
-                        Button {
-                            isOpen = false
-                        } label: {
-                            Text("Continue to Volume")
-                                .foregroundColor(Color.volume.orange)
-                                .font(.helveticaBold(size: 16))
+                            VStack {
+                                Image("volume-logo")
+                                    .resizable()
+                                    .frame(width: 245, height: 75)
+                                Text("Stay updated with Cornell student publications, all in one place")
+                                    .font(.begumRegular(size: 16))
+                            }
+                            .padding(.top, 200)
+                            Button {
+                                isOpen = false
+                            } label: {
+                                Text("Continue to Volume")
+                                    .foregroundColor(Color.volume.orange)
+                                    .font(.helveticaBold(size: 16))
+                            }
+                            .frame(width: 184, height: 45, alignment: .center)
+                            .overlay(RoundedRectangle(cornerRadius: 26)
+                            .stroke(Color.volume.orange, lineWidth: 2))
+                            .padding(.top, 200)
+                            Spacer()
                         }
-                        .frame(width: 184, height: 45, alignment: .center)
-                        .overlay(RoundedRectangle(cornerRadius: 26)
-                        .stroke(Color.volume.orange, lineWidth: 2))
-                        .padding(.top, 200)
-                        Spacer()
                     }
+                    .padding([.leading, .trailing], 50)
                 }
-                .padding([.leading, .trailing], 50)
+                .tabViewStyle(PageTabViewStyle())
             }
-            .tabViewStyle(PageTabViewStyle())
         }
-        .onAppear() {
+        .onAppear {
             UIPageControl.appearance().currentPageIndicatorTintColor = .gray
             UIPageControl.appearance().pageIndicatorTintColor = .lightGray
+            
+            if let uuid = userData.uuid {
+                fetchWeeklyDebrief(for: uuid)
+            }
+            
         }
-        .onDisappear() {
+        .onDisappear {
             UIPageControl.appearance().currentPageIndicatorTintColor = nil
             UIPageControl.appearance().pageIndicatorTintColor = nil
         }
     }
+    
+    
+    // MARK: - Copying from BrowserView
+    
```

Comment:
Delete new lines at beginning
```suggestion
    
    // MARK: - Copying from BrowserView
    
```

Reply:
The rest of AppDev projects wouldn't have one at the end either, but it looks like that's the style convention here.